docs/source/channel_update_tutorial.rst

```diff
@@ -489,6 +492,15 @@ peer to the channel:
 
   peer channel join -b mychannel.block
 
+To join by a snapshot, follow the instruction in [Taking a snapshot](./peer_ledger_snapshot.html#taking-a-snapshot)
+to take a snapshot on an existing peer. The snapshot should be taken after Org3 has been added to ``mychannel``
+to ensure that the snapshot contains the updated channel configuration including Org3.
+Locate the snapshot directory, copy it to the filesystem of the new Org3 peer, and issue the
+`peer channel joinbysnapshot` command using the path to the snapshot on your file system.
```

Comment:
This link is broken. It's formatted as a markdown link when this is an RST doc. This link *should* work, but we don't often use this type of link, so let's make sure to check it after it's updated: 

`Taking a snapshot <peer_ledger_snapshot.html#taking-a-snapshot>`_

Also you need double backtics behind a command in RST, so ``peer channel joinbysnapshot``.

Reply:
Thank you @joealewine. Done.